<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagePositionUI</class>
 <widget class="QWidget" name="ImagePositionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>178</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>314</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="framePosition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>130</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="glAlign">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="2" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QComboBox" name="cbPosition">
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>67</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>67</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choix du motif de fond&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
  padding: 1px -1px 1px 4px;

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QComboBox::drop-down {
	width: 0px;
}

QComboBox:hover{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
}


 QComboBox QAbstractItemView {
     selection-background-color: lightgray;
 }</string>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <property name="iconSize">
         <size>
          <width>52</width>
          <height>52</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/top_left</normaloff>:/images/aligment/top_left</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/top_center</normaloff>:/images/aligment/top_center</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/top_right</normaloff>:/images/aligment/top_right</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/center_left</normaloff>:/images/aligment/center_left</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/center_center</normaloff>:/images/aligment/center_center</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/center_right</normaloff>:/images/aligment/center_right</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/down_left</normaloff>:/images/aligment/down_left</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/down_center</normaloff>:/images/aligment/down_center</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/down_right</normaloff>:/images/aligment/down_right</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/aligment/align_perso</normaloff>:/images/aligment/align_perso</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="hsPositionH">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="vsPositionV">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="laH">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
        </property>
        <property name="text">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="laPosition">
        <property name="styleSheet">
         <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
    font:  bold 10pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
    font:  bold 10pt Calibri;
 }
</string>
        </property>
        <property name="text">
         <string>Alignement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="laV">
        <property name="styleSheet">
         <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
        </property>
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: rgb(0,106,255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameAdjust">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vlAdjust">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QComboBox" name="cbAdjust">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ajustement de l'image de fond&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid rgb(0,106,255);
    border-radius: 3px;
    padding: 1px 1px 1px -3px;

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);

    font:  10pt Calibri;
	color : rgb(150,150,150);
}


QComboBox:enabled {

	color : rgb(0,106,255);
}


QComboBox::drop-down {
	width: 0px;
}

QComboBox:hover{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
}

 QComboBox QAbstractItemView {
     selection-background-color: lightgray;
 }</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <property name="iconSize">
         <size>
          <width>72</width>
          <height>46</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Centrer</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/adjust/center_bg_image</normaloff>:/images/adjust/center_bg_image</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Etirer</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/adjust/extend_bg_image</normaloff>:/images/adjust/extend_bg_image</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Remplir</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/adjust/fill_bg_image</normaloff>:/images/adjust/fill_bg_image</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ajuster</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/adjust/adjust_bg_image</normaloff>:/images/adjust/adjust_bg_image</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mosaïque</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/adjust/mosaic_bg_image</normaloff>:/images/adjust/mosaic_bg_image</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="laScale">
        <property name="styleSheet">
         <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
     font:  9pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
     font:  9pt Calibri;
 }
</string>
        </property>
        <property name="text">
         <string>Facteur de taille de la photo
(Centrer, Remplir,Mosïque)</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QDoubleSpinBox" name="dsbScale">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="laAdjust">
        <property name="styleSheet">
         <string notr="true">

QWidget {
  
	color : rgb(0,106,255);
    font:  bold 10pt Calibri;


 }


QWidget:!enabled {
  
	color : rgb(150,150,150);
    font:  bold 10pt Calibri;
 }
</string>
        </property>
        <property name="text">
         <string>Ajustement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
